src/pages/Staking/index.tsx

```diff
@@ -870,16 +900,24 @@ const Staking = (): JSX.Element => {
                 value={renderUnlockDateLabel()}
                 tooltip='Your staked amount will be locked until this date.' />
             )}
+            <InformationUI
+              label={t('staking_page.new_vote_governance_token_gained', {
+                voteGovernanceTokenSymbol: VOTE_GOVERNANCE_TOKEN_SYMBOL
+              })}
+              value={renderNewVoteGovernanceTokenGainedLabel()}
+              tooltip={t('staking_page.the_increase_in_your_vote_governance_token_balance', {
+                voteGovernanceTokenSymbol: VOTE_GOVERNANCE_TOKEN_SYMBOL
+              })} />
             {votingBalanceGreaterThanZero && (
               <InformationUI
                 label='New total Stake'
-                value={`${renderNewTotalStakeLabel()} ${VOTE_GOVERNANCE_TOKEN_SYMBOL}`}
-                tooltip='Your total stake after this transaction.' />
+                value={`${renderNewTotalStakeLabel()}`}
+                tooltip='Your total stake after this transaction' />
```

Comment:
I think we should handle this message with i18n package just like all other tooltip messages.